src/common/token-swap/index.js

```diff
@@ -195,20 +195,25 @@ export const tokenSwapSelectors = {
 	selectTransaction: state => tokenSwapSelectors.selectRoot(state).transaction,
 	selectFee: state => {
 		const transaction = tokenSwapSelectors.selectTransaction(state);
-		const totleFee =
-			transaction && transaction.summary[0].totleFee
-				? +transaction.summary[0].totleFee.amount
-				: 0;
-		const partnerFee =
-			transaction && transaction.summary[0].partnerFee
-				? +transaction.summary[0].partnerFee.amount
-				: 0;
-		const total = transaction ? totleFee + partnerFee : 0;
+		let totleFee = 0;
+		let partnerFee = 0;
+		if (transaction) {
+			totleFee = transaction.summary.reduce((acc, curr) => acc + curr.totleFee.amount, 0);
+			partnerFee = transaction.summary.reduce((acc, curr) => acc + curr.partnerFee.amount, 0);
```

Comment:
better to combine the into one reduce unless the array traversal time is negligible 
```
{totalFee, partnerFee} = transactions.summary.reduce((acc, curr) => {
   acc.totalFee += curr.totleFee.amount;
  acc.partnerFee += curr.partnerFee;
  return acc;

}, {totalFee: 0, partnerFee: 0})

```

Reply:
This only to support future swap features.
Currently transactions.summary array is always 1 element.